<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>MakeLayout</class>
  <widget class="QWidget" name="MakeLayout">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>768</width>
        <height>642</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>KScope - Make</string>
    </property>
    <layout class="QVBoxLayout">
      <item>
        <layout class="QGridLayout">
          <item row="0" column="0">
            <widget class="QLabel" name="textLabel1">
              <property name="text">
                <string>Root Directory:</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="1" column="1">
            <widget class="KLineEdit" name="m_pMakeCommand">
              <property name="sizePolicy">
                <sizepolicy>
                  <hsizetype>7</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="clearButtonShown">
                <bool>true</bool>
              </property>
              <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
              </property>
            </widget>
          </item>
          <item row="0" column="1">
            <widget class="KUrlRequester" name="m_pRootURL"/>
          </item>
          <item row="1" column="0">
            <widget class="QLabel" name="textLabel2">
              <property name="text">
                <string>Command:</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <widget class="QTabWidget" name="tabWidget2">
          <widget class="QWidget" name="tab">
            <attribute name="title">
              <string>Output</string>
            </attribute>
            <layout class="QVBoxLayout">
              <item>
                <widget class="KTextBrowser" name="m_pOutputBrowser"/>
              </item>
            </layout>
          </widget>
          <widget class="QWidget" name="tab">
            <attribute name="title">
              <string>Errors a&amp;nd Warnings</string>
            </attribute>
            <layout class="QVBoxLayout">
              <item>
                <widget class="QueryView" name="m_pErrorView"/>
              </item>
            </layout>
          </widget>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <item>
            <spacer name="spacer1">
              <property name="sizeHint">
                <size>
                  <width>520</width>
                  <height>21</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="m_pMakeButton">
              <property name="text">
                <string>&amp;Make</string>
              </property>
              <property name="shortcut">
                <string>Alt+M</string>
              </property>
              <property name="default">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="m_pStopButton">
              <property name="enabled">
                <bool>false</bool>
              </property>
              <property name="text">
                <string>&amp;Stop</string>
              </property>
              <property name="shortcut">
                <string>Alt+S</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="m_pCloseButton">
              <property name="text">
                <string>&amp;Close</string>
              </property>
              <property name="shortcut">
                <string>Alt+C</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <customwidgets>
    <customwidget>
      <class>QueryView</class>
      <extends>QWidget</extends>
      <header location="local">queryview.h</header>
      <sizehint>
        <width>-1</width>
        <height>-1</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>5</hordata>
        <verdata>5</verdata>
      </sizepolicy>
      <pixmap>image0</pixmap>
    </customwidget>
  </customwidgets>
  <tabstops>
    <tabstop>m_pRootURL</tabstop>
    <tabstop>m_pMakeCommand</tabstop>
    <tabstop>m_pMakeButton</tabstop>
    <tabstop>m_pStopButton</tabstop>
    <tabstop>m_pCloseButton</tabstop>
  </tabstops>
  <includes>
    <include location="local">klineedit.h</include>
    <include location="local">kurlrequester.h</include>
    <include location="local">kpushbutton.h</include>
    <include location="local">ktextbrowser.h</include>
    <include location="local">queryview.h</include>
  </includes>
  <connections>
    <connection>
      <sender>m_pCloseButton</sender>
      <signal>clicked()</signal>
      <receiver>MakeLayout</receiver>
      <slot>close()</slot>
    </connection>
    <connection>
      <sender>m_pMakeButton</sender>
      <signal>clicked()</signal>
      <receiver>MakeLayout</receiver>
      <slot>slotMake()</slot>
    </connection>
    <connection>
      <sender>m_pStopButton</sender>
      <signal>clicked()</signal>
      <receiver>MakeLayout</receiver>
      <slot>slotStop()</slot>
    </connection>
    <connection>
      <sender>m_pMakeCommand</sender>
      <signal>returnPressed(const QString)</signal>
      <receiver>MakeLayout</receiver>
      <slot>slotMake(const QString)</slot>
    </connection>
  </connections>
</ui>
